setup.cfg

```diff
@@ -20,7 +20,7 @@ classifiers =
 
 [options]
 packages = find:
-python_requires = >= 3.6
+python_requires = >=3.6, !=3.9.*, !=3.10.*
```

Comment:
Can consider this instead.
```python
python_requires = >=3.6, <3.9
```
This gives a similar error and potentially future proof too.
```
ERROR: Package 'peekingduck' requires a different Python: 3.9.7 not in '<3.9,>=3.6'
```

Reply:
Good suggestion, I've updated accordingly.